tests/test_data.py

```diff
@@ -3,12 +3,14 @@
 import os
 import pytest
 import sys
+import requests
 
 sys.path.append('.')
 from simple.schema import *
 from astrodbkit2.astrodb import create_database, Database
 from sqlalchemy import except_, select, and_
 from . import REFERENCE_TABLES
+from scripts.ingests.utils import check_internet_connection
```

Comment:
Do you still need this import now that you moved the logic to scheduled_checks.py?